File: Domain/DebitSecurity.Domain/Entities/Debit.cs
using System;
using System.Collections.Generic;

namespace DebitSecurity.Domain.Entities
{
    public class Debit: BaseEntity
    {
        public string SecurityNumber { get; set; }
        public double Interest { get; set; }
        public double Penalty { get; set; }
        public Guid PersonId { get; set; }
        public Person Person { get; set; }
        public IList<Installment> Installments { get; set; }
    }
}

File: App/DebitSecurityAPP/src/app/models/debit.ts
import { Installment } from "./installment";

export class Debit {
  Penalty!: number;
  Interest!: number;
  NumberInstalments!: number;
  OriginalValue!: number;
  DaysOverdue!: number;
  ActualValue!: number;
  Installments!: Installment[];
  DocumentId!: number;
  Id!: number;
}
